pipeline_dp/dp_engine.py

```diff
@@ -98,28 +97,33 @@ def aggregate(self, col, params: AggregateParams,
         return col
 
     def _drop_not_public_partitions(self, col, public_partitions,
-                                    data_extractors):
-        return self._ops.filter_by_key(col, public_partitions, data_extractors,
-                                       "Filtering out non-public partitions")
+                                    data_extractors: DataExtractors):
+        """"""
```

Comment:
Nit: Remove empty comment line

Reply:
Thanks, I wanted to add a comment and forgot to do it. I've added a comment